specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json

```diff
@@ -100,6 +100,10 @@
           "type": "integer",
           "minimum": 1
         },
+        "dataFlowProperties": {
```

Comment:
adding new writable properties is a breaking change and must be done in a new API version. But I think for datafactory an exception was given before. SO I am fine signing off on this. But please ensure customer cant unintentionally overwrite the goal state of the resource when making an update from an older SDK.